templates/common/upgrade/upgradeall.go

```diff
@@ -104,7 +104,8 @@ func UpgradeAll(ctx context.Context, p *Params) *Result {
 		}
 		logger.InfoContext(ctx, "beginning upgrade of manifest",
 			"manifest", absManifestPath)
-		result, err := upgrade(ctx, p, absManifestPath)
+		manifest := manifests[manifestPath]
```

Comment:
should we check for non-existence here or does that happen later?

Reply:
I just added a comment to make this more clear. We're guaranteed that `manifests` and `sorted` are one-to-one, so this lookup is guaranteed to succeed.